app/src/main/java/com/onix/internship/ui/game/TicTacToeBoard.kt

```diff
@@ -0,0 +1,213 @@
+package com.onix.internship.ui.game
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.TypedArray
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import com.onix.internship.R
+import kotlin.math.ceil
+
+class TicTacToeBoard(context: Context, attrs: AttributeSet?) : View(context, attrs) {
+
+    private val size = 0.2f
+    private var winningLine = false
+    private val paint = Paint()
+    private var cellSize = width / 3
+    private val game = GameViewModel()
```

Comment:
game should be stored inside viewmodel and view just render model on scren and pass user input into viewmodel

Reply:
done